public/examples.html

```diff
@@ -2,26 +2,23 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Lighthouse Score Badge Previews</title>
-  <style>
-
-  </style>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
 </head>
 <body>
 
-<p><img src="/?score=35"> <code>&lt;img src="https://lighthouse-badge.appspot.com?score=35"></code></p>
-<p><img src="/?score=65"> <code>&lt;img src="https://lighthouse-badge.appspot.com?score=65"></code></p>
-<p><img src="/?score=95"> <code>&lt;img src="https://lighthouse-badge.appspot.com?score=95"></code></p>
+<p><img src="/?score=35"> <code>&lt;img src="https://lighthouse-badge.appspot.com?score=35" alt="Lighthouse score: 35/100"></code></p>
+<p><img src="/?score=65"> <code>&lt;img src="https://lighthouse-badge.appspot.com?score=65" alt="Lighthouse score: 65/100"></code></p>
+<p><img src="/?score=95"> <code>&lt;img src="https://lighthouse-badge.appspot.com?score=95" alt="Lighthouse score: 95/100"></code></p>
 
-<p><img src="/?score=95&category=PWA"><code>&lt;img src="https://lighthouse-badge.appspot.com?score=95&category=PWA"></code></p>
+<p><img src="/?score=95&amp;category=PWA"><code>&lt;img src="https://lighthouse-badge.appspot.com?score=95&category=PWA" alt="Lighthouse PWA score: 95/100"></code></p>
 
-<p><img src="/?score=35&compact"> <code>&lt;img src="https://lighthouse-badge.appspot.com?score=35&compact"></code></p>
-<p></p><img src="/?score=65&compact"> <code>&lt;img src="https://lighthouse-badge.appspot.com?score=65&compact"></code></p>
-<p><img src="/?score=95&compact"> <code>&lt;img src="https://lighthouse-badge.appspot.com?score=95&compact"></code></p>
+<p><img src="/?score=35&amp;compact"> <code>&lt;img src="https://lighthouse-badge.appspot.com?score=35&amp;amp;compact" alt="Lighthouse score: 35/100"></code></p>
```

Comment:
The `&` shouldnt be encoded in these urls.

Reply:
They don't need to be in this case.

But it's generally easier and less error-prone to consistently encode ampersands in HTML, unless you like to remember the exact details of ["ambiguous ampersands"](https://mathiasbynens.be/notes/ambiguous-ampersands) (I sure don't).